src/test/java/itest/TargetEventsGetIT.java

```diff
@@ -80,10 +80,30 @@ public void testGetTargetEventsReturnsListOfEvents() throws Exception {
     }
 
     @Test
-    public void testGetTargetEventsSearchReturnsRequestedEvents() throws Exception {
+    public void testGetTargetEventsV2WithNoQueryReturnsListOfEvents() throws Exception {
+        CompletableFuture<JsonArray> getResponse = new CompletableFuture<>();
+        webClient
+                .get(SEARCH_REQ_URL)
+                .send(
+                        ar -> {
+                            if (assertRequestStatus(ar, getResponse)) {
+                                MatcherAssert.assertThat(
+                                        ar.result().statusCode(), Matchers.equalTo(200));
+                                MatcherAssert.assertThat(
+                                        ar.result().getHeader(HttpHeaders.CONTENT_TYPE.toString()),
+                                        Matchers.equalTo(HttpMimeType.JSON.mime()));
+                                getResponse.complete(ar.result().bodyAsJsonArray());
+                            }
+                        });
+
+        MatcherAssert.assertThat(getResponse.get().size(), Matchers.greaterThan(0));
+    }
+
+    @Test
+    public void testGetTargetEventsV2WithQueryReturnsRequestedEvents() throws Exception {
         CompletableFuture<JsonObject> getResponse = new CompletableFuture<>();
         webClient
-                .get(String.format("%s/WebServerRequest", SEARCH_REQ_URL))
+                .get(String.format("%s?q=WebServerRequest", SEARCH_REQ_URL))
```

Comment:
Since the request format changed, should `HTTP_API.md` be updated?

Reply:
Good idea.